README.md

```diff
@@ -27,16 +27,21 @@ import (
 )
 
 func main() {
-
-    // Set TERMII_API_KEY, TERMII_URL and TERMII_SENDER_ID as environment variables
     // Ensure TERMII_URL ends with '/' i.e 'https://termii.com/'
+   var httpClient *http.Client
+
+	termiiConfig := termii.Config{
+		APIKey:   os.Getenv("TERMII_API_KEY"),
+		BaseURL:  os.Getenv("TERMII_BASE_URL"),
+		SenderID: os.Getenv("TERMII_SENDER_ID"),
+	}
+
+	client := termii.NewClient(termiiConfig, httpClient)
 
     req := termii.AutoGeneratedMessageRequest{
         To:  "2347066554433",
         Sms: "Hello from termii", // You do not need to set the API Key in this request struct
     }
-
-    client := termii.NewClient()
```

Comment:
Leave the existing slow. Simply show there is an alternative way of initialing the client

Reply:
Sorry, it took this long to come back to this.
If I understand you clearly from what you said in your comment, you want me to leave the existing flow and show another way to init the client. I think it will be better if there is only one way to init the client. Furthermore, having two ways of initialing will cause more codes to be added with some modifications to each methods to watch for whichever `NewClient` method is being initialized. 
I am suggesting that, if you think its better off to have it the way it is right now(watching out for certain keys in the .env), then let it follow that path and if you think giving the developers that flexibly to set these up themseleve by passing the config values into the `NewClient` method then let the modification be made for this package to use the init method.

Reply:
We can't change the existing way of initializing the client because it is a breaking change